client/client-multi/client-c/CMakeLists.txt

```diff
@@ -147,16 +154,12 @@ add_library(kaac SHARED ./src $<TARGET_OBJECTS:kaac_o>)
 target_link_libraries(kaac ${KAA_THIRDPARTY_LIBRARIES})
 
 # Sets paths Kaa libraries will be installed in.
-install(TARGETS kaac kaac_s
-        LIBRARY DESTINATION lib
-        ARCHIVE DESTINATION lib
-        RUNTIME DESTINATION lib
-)
-
-install(DIRECTORY src/ 
-        DESTINATION include/kaac
-        FILES_MATCHING PATTERN *.h
-)
+message(STATUS "KAA WILL BE INSTALLED TO  ${KAA_INSTALL_PATH} ")
```

Comment:
Please, use one version of the message() function all around the file - either with the mode (e.g. STATUS) or not.